File: src/statesman/commands/QuitCommand.java
package statesman.commands;

public class QuitCommand implements Command {

    public static final String ID = "quit";
    
    public QuitCommand() {
    }

    @Override
    public Command createInstance(String[] arguments) {
        return new QuitCommand();
    }

    @Override
    public void execute() {
        System.exit(0);
    }

}

File: src/Commands/QuitCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Statesman.Commands
{
    public class QuitCommand : Command
    {
        public static readonly string ID = "quit";

        public QuitCommand()
        {
        }

        public override Command createInstance(string[] arguments)
        {
            return new QuitCommand();
        }

        public override void execute()
        {
            Environment.Exit(0);
        }
    }
}
